.github/workflows/e2e-tests.yml

```diff
@@ -62,7 +62,7 @@ jobs:
           cluster-name: ${{ github.repository_owner }}-ghactions-cluster
       - name: "Run all end-to-end tests"
         run: |
-          make --directory e2e-tests tests upgrade.bats
+          make --directory e2e-tests tests upgrade.bats audit-scanner-installation.bats
```

Comment:
Note that there's `audit-scanner-installation.bats` which tests the installation, and `audit-scanner.bats` added on this PR, which is not yet run if I understand it correctly.

Reply:
thanks for the explanation! resolving